manifests/tunnel.pp

```diff
@@ -7,37 +7,29 @@
   ensure_resource('package', 'spiped', {'ensure' => 'present'})
   ensure_resource('file', '/etc/spiped', {'ensure' => 'directory'})
 
-  $unitfile = "/lib/systemd/system/spiped-${title}.service"
   $keyfile = "/etc/spiped/${title}.key"
 
-  file {
-    $unitfile:
-      owner   => root,
-      group   => root,
-      content => template('spiped/service.erb'),
-      require => File[$keyfile];
+  file { $keyfile:
+    owner     => root,
+    group     => root,
+    mode      => '0600',
+    show_diff => false,
+    content   => $secret,
+  }
 
-    $keyfile:
-      owner     => root,
-      group     => root,
-      mode      => '0600',
-      show_diff => false,
-      content   => $secret,
-      require   => File['/etc/spiped'];
+  # Previous versions created a unit file here.
+  file { "/lib/systemd/system/spiped-${title}.service":
```

Comment:
I am not sure if this resource provides much value. unit files in /etc/ overwrite the ones in /lib. So we don't need to remove it, but the presence might confuse people.